activiti-api-impl/activiti-api-process-model-impl/pom.xml

```diff
@@ -19,7 +19,7 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
   <parent>
-    <groupId>org.activiti.api</groupId>
+    <groupId>org.activiti</groupId>
```

Comment:
@Salaboy this make me think about the discussion we had earlier about naming group ids based on the repository where they are.
I don't have a strong feeling about it, but IMO it would make sense that the model interface and the model implementation have the same group id. If we follow the repository approach they will be different:

```xml
     <dependency>
       <groupId>org.activiti.api</groupId>
       <artifactId>activiti-api-process-model</artifactId>
     </dependency>
```
vs
```xml
     <dependency>
       <groupId>org.activiti</groupId>
       <artifactId>activiti-api-process-model-impl</artifactId>
     </dependency>
```

Reply:
Yes, they are different in here.. I do believe that we get more by having different groupIds that having them aligned. I will advice to start with this approach to see if bother us or if it works well. I cannot think of any downside of not having the interface and impl under a different groupId, for me it makes it clear that one is the implementation and the other the interface.